deepmd/dpmodel/descriptor/dpa2.py

```diff
@@ -516,42 +624,8 @@ def serialize(self) -> dict:
             "type": "dpa2",
             "@version": 1,
             "ntypes": self.ntypes,
-            "repinit_rcut": repinit.rcut,
-            "repinit_rcut_smth": repinit.rcut_smth,
-            "repinit_nsel": repinit.sel,
-            "repformer_rcut": repformers.rcut,
-            "repformer_rcut_smth": repformers.rcut_smth,
-            "repformer_nsel": repformers.sel,
-            "repinit_neuron": repinit.neuron,
-            "repinit_axis_neuron": repinit.axis_neuron,
-            "repinit_tebd_dim": repinit.tebd_dim,
-            "repinit_tebd_input_mode": repinit.tebd_input_mode,
-            "repinit_set_davg_zero": repinit.set_davg_zero,
-            "repinit_activation_function": repinit.activation_function,
-            "repinit_resnet_dt": repinit.resnet_dt,
-            "repinit_type_one_side": repinit.type_one_side,
-            "repformer_nlayers": repformers.nlayers,
-            "repformer_g1_dim": repformers.g1_dim,
-            "repformer_g2_dim": repformers.g2_dim,
-            "repformer_axis_neuron": repformers.axis_neuron,
-            "repformer_direct_dist": repformers.direct_dist,
-            "repformer_update_g1_has_conv": repformers.update_g1_has_conv,
-            "repformer_update_g1_has_drrd": repformers.update_g1_has_drrd,
-            "repformer_update_g1_has_grrg": repformers.update_g1_has_grrg,
-            "repformer_update_g1_has_attn": repformers.update_g1_has_attn,
-            "repformer_update_g2_has_g1g1": repformers.update_g2_has_g1g1,
-            "repformer_update_g2_has_attn": repformers.update_g2_has_attn,
-            "repformer_update_h2": repformers.update_h2,
-            "repformer_attn1_hidden": repformers.attn1_hidden,
-            "repformer_attn1_nhead": repformers.attn1_nhead,
-            "repformer_attn2_hidden": repformers.attn2_hidden,
-            "repformer_attn2_nhead": repformers.attn2_nhead,
-            "repformer_attn2_has_gate": repformers.attn2_has_gate,
-            "repformer_activation_function": repformers.activation_function,
-            "repformer_update_style": repformers.update_style,
-            "repformer_set_davg_zero": repformers.set_davg_zero,
-            "repformer_trainable_ln": repformers.trainable_ln,
-            "repformer_ln_eps": repformers.ln_eps,
+            "repinit": self.repinit_args.serialize(),
+            "repformer": self.repformer_args.serialize(),
```

Comment:
we may want to name them by repinit_args and repformer_args?

Reply:
I agree, https://github.com/deepmodeling/deepmd-kit/pull/3768/commits/7936f9d6357db2bc6aa4512ce9312952c386f83c changes their names in serialized dict. But in input.json, I think users prefer to use "repinit": {} and "repformer": {} for convenience, so I kept them.